src/index.js

```diff
@@ -66,7 +70,7 @@ const MicroModal = (() => {
     }
 
     onClick (event) {
-      if (event.target.hasAttribute('data-micromodal-close')) this.closeModal()
```

Comment:
After reviewing this, I understood what the problem was. Essentially it's a syntax error I made when writing the code. I guess this is one of the shortfalls of too much shorthand.

Simply changing

```js
if (event.target.hasAttribute('data-micromodal-close')) this.closeModal()
event.preventDefault()
```

to 

```js
if (event.target.hasAttribute('data-micromodal-close')) {
    this.closeModal()
    event.preventDefault()
}
```

will make it work perfectly. No other change is required really..

Reply:
Updated PR.